tests/AutoReview/FixerFactoryTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\AutoReview;
+
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerFactory;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @coversNothing
+ * @group auto-review
+ */
+final class FixerFactoryTest extends \PHPUnit_Framework_TestCase
```

Comment:
I think this can be merged in `FixerTest` (like before) as both classes do general tests on all fixers. Otherwise class name should be changed as it does no test `FixerFactory` specifically.